src/goto-instrument/reachability_slicer.cpp

```diff
@@ -20,41 +24,46 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "full_slicer_class.h"
 #include "reachability_slicer_class.h"
 
-void reachability_slicert::fixedpoint_assertions(
+/// Perform backwards breadth-first-search of the control-flow graph of the
+/// goto program, starting from the criterion. Set reaches_assertion to true
+/// for every instruction visited.
+/// \param criterion the criterion we are trying to hit
+/// \param is_threaded TODO
```

Comment:
`is_threadedt` overapproximates the set of instructions that might be executed concurrently. These cannot be safely sliced away because we lack information on the multithreaded control flow here (synchronisation).